app/models/employee.rb

```diff
@@ -0,0 +1,2 @@
+class Employee < ApplicationRecord
```

Comment:
I know, I didn't say anything related to validations, but I think we can validate the presence of name here, as a required attribute, but no worries, you can add it in the next PR. just a suggestion for the future